.travis.yml

```diff
@@ -12,20 +12,21 @@ matrix:
       env:
         CXX: clang++
         CC: clang
+
     - os: osx
       osx_image: xcode10.1
       language: generic
       python: 3.6
       env:
         CXX: clang++
         CC: clang
-     - os: osx
-       osx_image: xcode10.1
-       language: generic
-       python: 2.7
-       env:
-         CXX: g++
-         CC: gcc
+    - os: osx
+      osx_image: xcode10.1
+      language: generic
+      python: 2.7
+      env:
+        CXX: g++
+        CC: gcc
```

Comment:
what is providing gcc?  is it brew? I'm not sure this is needed in that case, as it should be covered by the linux plans, which run a lot quicker

Reply:
It does not seem to be brew: /Applications/Xcode-10.1.app/Contents/Developer/usr/bin/g++
Should I still remove it ?

Reply:
I suppose it probably runs in parallel, so it won't reduce the build time.  Don't worry about it.